### PR TITLE
Add Post GraphQL endpoints

### PR DESCRIPTION
**Pull request by tmock12**
I implemented a GraphQL endpoint only because I was interested in playing around with it. Its working and we could theoretically replace the existing rest API with this if we want. We could also add to it if we feel like we want a full-fledged API. 

Just thought others may like to take a look at how easy it was. Feel free to merge, close or discuss.

If you visit http://localhost:4000/apiv2/graphiql you get a nice graphical interface for playing around with it and trying different filters:

screen shot 2018-03-20 at 5 35 40 pm
screen shot 2018-03-20 at 5 28 23 pm

### CODE REVIEW

**Comment by chriserin** [comment]
@tmock12 how are you feeling about this?  Is it still something you'd like to see in the tilex ?

**Comment by dkarter** [comment]
Damn, I could really use this PR in my new blog - would like to link to all my old posts :) Any chance we can revisit that?

**Comment by tmock12** [comment]
If others find it useful then I wouldn't oppose. The reason I closed it is because I was the only one that wanted it. It probably wouldn't take much to update and cleanup. 

Actually if we did this, and I could get a guest account (🙏 ) then I would  gladly use `til.hashrocket.com`.

**Comment by greis** [comment]
@dkarter @tmock12  I was pairing with @jwworth this morning and he told me to look at this thread since I have been using Absinthe in my side project. The code looks good but I think it would be nice to upgrade absinthe to the latest version 1.6.3

**Comment by tmock12** [comment]
@greis yea as I mentioned, there's some cleanup and updating to be done as its from early 2018 and was a POC, but shouldn't be too much. It's more a question of "_do people want this?_"

File: lib/tilex_web/schema/post_types.ex [hunk 1]
```diff
@@ -0,0 +1,24 @@
+defmodule TilexWeb.Schema.PostTypes do
+  use Absinthe.Schema.Notation
+  use Absinthe.Ecto, repo: Tilex.Repo
+
+  object :post do
+    field(:title, :string)
+    field(:body, :string)
+    field(:slug, :string)
+    field(:likes, :integer)
+    field(:developer, :developer, resolve: assoc(:developer))
```
Comment (greis) [inline lib/tilex_web/schema/post_types.ex:10]: logic here for `developer` and `channel` should probably change to use DataLoaders https://hexdocs.pm/absinthe/dataloader.html

**Comment by greis** [comment]
@tmock12 It sounds like @dkarter wants it, so I'd say yes. @jwworth any thoughts on this?

File: lib/tilex_web/schema/post_types.ex (hunk 1 above)
Comment (tmock12) [reply lib/tilex_web/schema/post_types.ex:10]: yea, its definitely not in s state that should be reviewed at the moment. I don't even thing absinthe_ecto is around anymore.

**Pull request closed without merging**